FrontEnd/src/components/AboutPage/MeetTheTeam/MemberInfoCard/MemberInfoCard.tsx

```diff
@@ -48,7 +47,7 @@ const MemberInfoCard: FunctionComponent<MemberInfoCardProps> = ({
         </Row>
         <div className={styles.caption}>
           {typeof quote === 'string' ? (
-            <Body2>"{quote}"</Body2>
+            <Body2>{'"' + quote + '"'}</Body2>
```

Comment:
ah wait fuck I think you were right before LOL my bad, I didn't have the code coloring on here so I just used my eyesight... whichever one works correctly you can use

Reply:
LOL no worries, I can fix it up real quick